modules/python3/bindings/src/pyvolume.cpp

```diff
@@ -34,8 +34,10 @@
 #include <inviwopy/inviwopy.h>
 #include <inviwopy/pynetwork.h>
 #include <inviwopy/pyglmtypes.h>
+#include <inviwopy/pyimage.h>  // for the opaque swizzlemask
```

Comment:
still opaque? That was removed from pyimage.h.

Reply:
still there